spec/models/form/sales/subsections/household_needs_spec copy.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
```

Comment:
added this file in even though it's not technically related to this ticket as it was efficient to do it while I'm making another similar spec